index.html

```diff
@@ -1,227 +1,286 @@
-              <li>CSS</li>
-              <li>Javascript</li>
-              <li>HTML</li>
-            </ul>
-            <a href="#" class="works-button">See Project</a>
+        </section>
+      </header>
+      </div>
+
+      <section class="recent-works">
+        <div class="title">
+          <h2>My Recent Works</h2>
+          <div class="bar">
+            <img src="assets/images/Indicator.png" alt="Indicator" />
           </div>
         </div>
-        
-        <div class="works work4">
-          <div class="work-details"> 
-            <h3>Multi-Post Stories Gain+Glory</h3>  
-            <ul class="language-tags"> 
-              <li>Ruby on Rails</li>
-              <li>CSS</li>
-              <li>Javascript</li>
-              <li>HTML</li>
-            </ul>
-            <a href="#" class="works-button">See Project</a>
+
+        <div class="recent-works-cards">
+          <div class="works work1">
+            <div class="work-details">
+              <h3>Multi-Post Stories Gain+Glory</h3>
+              <ul class="language-tags">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <a href="#" class="works-button">See Project</a>
+            </div>
           </div>
-        </div>
-        
-        <div class="works work5">
-          <div class="work-details"> 
-            <h3>Multi-Post Stories Gain+Glory</h3> 
-            <ul class="language-tags"> 
-              <li>Ruby on Rails</li>
-              <li>CSS</li>
-              <li>Javascript</li>
-              <li>HTML</li>
-            </ul>
-            <a href="#" class="works-button">See Project</a>
+
+          <div class="works work2">
+            <div class="work-details">
+              <h3>Multi-Post Stories Gain+Glory</h3>
+              <ul class="language-tags">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <a href="#" class="works-button">See Project</a>
+            </div>
           </div>
-        </div>
-        
-        <div class="works work6">
-          <div class="work-details"> 
-            <h3>Multi-Post Stories Gain+Glory</h3> 
-            <ul class="language-tags"> 
-              <li>Ruby on Rails</li>
-              <li>CSS</li>
-              <li>Javascript</li>
-              <li>HTML</li>
-            </ul>
-            <a href="#" class="works-button">See Project</a>
+
+          <div class="works work3">
+            <div class="work-details">
+              <h3>Multi-Post Stories Gain+Glory</h3>
+              <ul class="language-tags">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <a href="#" class="works-button">See Project</a>
+            </div>
```

Comment:
- [ ] Great job rearranging your code, however, in the project summary you stated that only the title has no issue. Kindly do that for all of the required checks and if you found any issue please leave the link, or else no issue found.

![image](https://user-images.githubusercontent.com/59416136/155159888-aacf5574-4958-4585-abb1-588374128525.png)